app/src/main/res/values/strings.xml

```diff
@@ -13,11 +13,20 @@
         <item>HOD</item>
         <item>Teacher</item>
     </string-array>
+    <string-array name="profile_dropdown">
+        <item>Change Password</item>
+        <item>Profile</item>
+        <item>Logout</item>
+    </string-array>
+
+
 
```

Comment:
also, check now my new commit and upstream your branch.
As I'll make a partition for all the strings so arrange them according to that